CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to
 
 ## [Unreleased]
 
+## [0.1.3] - 2024-08-28
```

Comment:
I'd still add the content of the changelog as things are added but the version here becomes 

`## [0.1.3] - TBD`

Reply:
Thanks for the reminder, I've updated it here: fdbf498fde7a1501ec820268295abda300caa99f